internal/sanitize/node.go

```diff
@@ -116,6 +127,19 @@ func (n *Node) checkConditions(ctx context.Context, no *v1.Node) bool {
 	return ready
 }
 
+// checkMasterRole checks whether the node is a master node.
+func (n *Node) checkMasterRole(ctx context.Context, no *v1.Node) bool {
+	var isMaster bool = false
+
+	if role, ok := no.Labels["role"]; ok {
```

Comment:
I don't think this check is valid. I think the official label should be `node-role.kubernetes.io/master`. Also I believe there is proposal to rename this label to `node-role.kubernetes.io/control-plane` so we should define a couple of constants here so we can correctly check for today and future clusters.

Reply:
Changed